android/app/src/main/java/org/fossasia/openevent/core/auth/ChangePasswordActivityViewModel.java

```diff
@@ -75,7 +75,10 @@ public ChangePasswordActivityViewModel() {
         int id;
         try {
             id = JWTUtils.getIdentity(AuthUtil.getAuthorization());
-            compositeDisposable.add(APIClient.getOpenEventAPI().updateUser(User.builder().id(id).password(newPassword).build(), id)
+            User u = new User();
```

Comment:
Proper naming please - `user`

Reply:
user is already being used in a code below. Ill use userModel instead